core/modules/layout/tests/layout.test

```diff
@@ -1133,14 +1133,22 @@ class LayoutInterfaceTest extends BackdropWebTestCase {
     );
     $this->backdropPost('admin/structure/layouts/manage/' . $layout_name . '/configure', $edit, t('Save layout'));
 
-    $deleted_uuids = $default_layout->positions['sidebar'];
+    $orphaned_uuids = $default_layout->positions['sidebar'];
     layout_reset_caches();
     $layout = layout_load($layout_name);
-    $this->assertBlocksMatch($layout, $default_layout);
-    $this->assertFalse(isset($layout->positions['sidebar']), 'Sidebar region blocks removed from layout when changing layout to single column.');
+    $this->assertBlocksMatch($layout, $default_layout, $orphaned_uuids);
+    $this->assertFalse(isset($layout->positions['sidebar']), 'Sidebar region blocks moved out when changing layout to single column.');
 
-    $intersection = array_intersect(array_keys($layout->content), $deleted_uuids);
+    $intersection = array_intersect(array_keys($layout->content), $orphaned_uuids);
     $this->assertEqual($intersection, array(), 'The sidebar blocks have been removed from the layout content array.');
+    $default_layout_sidebar_content_regions_count = count($default_layout->positions['sidebar']) + count($default_layout->positions['content']);
```

Comment:
A code comment here saying whats happening would be nice;  took me a while to figure it out.

Reply:
Sure